ci-operator/templates/openshift/installer/cluster-launch-installer-openstack-e2e.yaml

```diff
@@ -80,15 +80,30 @@ objects:
     - name: cluster-profile
       secret:
         secretName: ${JOB_NAME_SAFE}-cluster-profile
+
     dnsPolicy: "None"
     dnsConfig:
       nameservers:
         - ${CI_DNS_IP}
+    initContainers:
+    - name: cli
+      image: ${IMAGE_TESTS}
+      volumeMounts:
+      - name: shared-tmp
+        mountPath: /tmp/shared
+      command:
+      - cp
+      - /usr/bin/openshift-tests
+      - /tmp/shared/openshift-tests
```

Comment:
Might as well just copy it into `/usr/bin/openshift-tests`, no?  We know the installer repo won't have a binary there, and it lets us leave `PATH` alone.

Reply:
Just reviewed the suggestions on slack and we need both openshift-tests and oc I think. So lets stick with /tmp/shared right?

Reply:
> Just reviewed the suggestions on slack and we need both openshift-tests and oc I think. So lets stick with /tmp/shared right?

Oh, I see, these are still getting into the `test` container via the shared volume.  Yeah, that makes sense.